tcms/issuetracker/azure_boards.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+from requests.auth import HTTPBasicAuth
+import requests
+
+from tcms.core.contrib.linkreference.models import LinkReference
+from tcms.issuetracker.base import IssueTrackerType, IntegrationThread
+
+
+class AzureAPI():
+    """
+        Azure Boards API interaction class.
+
+    """
+    def __init__(self, base_url=None, password=None):
+        self.api_version = "?api-version=6.0"
+        self.headers = {"Accept": 'application/json-patch+json',
+                        'Content-type': 'application/json-patch+json'}
+        self.auth = HTTPBasicAuth('apikey', password)
+        self.base_url = base_url + "/_apis/"
+
+    def get_issue(self, issue_id):
+        url = "{0}{1}{2}{3}".format(self.base_url, "wit/workitems/",
+                                    str(issue_id), self.api_version)
```

Comment:
when using `.format` you don't have to specifically cast the values to string. This is what format does.